.ci/install.sh

```diff
@@ -38,7 +38,10 @@ python3 -m pip install -U pytest-timeout
 python3 -m pip install pyroma
 
 if [[ $(uname) != CYGWIN* ]]; then
-    python3 -m pip install numpy
+    # NumPy doesn't support PyPy3.11
+    if [[ $GHA_PYTHON_VERSION != pypy3.11 ]]; then
+        python3 -m pip install numpy
+    fi
```

Comment:
From what I see, NumPy works with PyPy 3.11 on both macOS and Ubuntu - https://github.com/radarhere/Pillow/tree/refs/heads/numpy - https://github.com/radarhere/Pillow/actions/runs/13360699763

Reply:
Excellent, the previously-failing job passes on a re-run.

https://github.com/hugovk/Pillow/actions/runs/13203963591/job/37329242544

I've rebased the skip out :)